.github/workflows/platform.set-avm-github-issue-label.yml

```
@@ -0,0 +1,41 @@
+# Workflow for checking issues if correct labels are applied and corrects if necessary
+name: .Platform - Set GitHub issue label
+
+on:
+  schedule:
+    - cron: "30 2 * * *" # Every day at 2:30 am
```

I'd suggest a weekly run to start
```suggestion
    - cron: "30 2 * * Sun" # Every Sunday at 2:30 am
```